[2025-01-31T02:52:30.512+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T02:52:30.533+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_and_insert_csv_to_postgres_with_branch_adjusted.check_files_branch_task manual__2025-01-31T02:52:20.519782+00:00 [queued]>
[2025-01-31T02:52:30.542+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_and_insert_csv_to_postgres_with_branch_adjusted.check_files_branch_task manual__2025-01-31T02:52:20.519782+00:00 [queued]>
[2025-01-31T02:52:30.544+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-01-31T02:52:30.557+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): check_files_branch_task> on 2025-01-31 02:52:20.519782+00:00
[2025-01-31T02:52:30.568+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=124) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T02:52:30.569+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_and_insert_csv_to_postgres_with_branch_adjusted', 'check_files_branch_task', 'manual__2025-01-31T02:52:20.519782+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/read_file_updated.py', '--cfg-path', '/tmp/tmpkbypidr0']
[2025-01-31T02:52:30.570+0000] {standard_task_runner.py:72} INFO - Started process 126 to run task
[2025-01-31T02:52:30.570+0000] {standard_task_runner.py:105} INFO - Job 6: Subtask check_files_branch_task
[2025-01-31T02:52:30.617+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_and_insert_csv_to_postgres_with_branch_adjusted.check_files_branch_task manual__2025-01-31T02:52:20.519782+00:00 [running]> on host e2498a6c8083
[2025-01-31T02:52:30.682+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_and_insert_csv_to_postgres_with_branch_adjusted' AIRFLOW_CTX_TASK_ID='check_files_branch_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T02:52:20.519782+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-31T02:52:20.519782+00:00'
[2025-01-31T02:52:30.684+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-31T02:52:30.685+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-31T02:52:30.685+0000] {logging_mixin.py:190} INFO - Current task name:check_files_branch_task state:running start_date:2025-01-31 02:52:30.534971+00:00
[2025-01-31T02:52:30.686+0000] {logging_mixin.py:190} INFO - Dag name:validate_and_insert_csv_to_postgres_with_branch_adjusted and current dag run status:running
[2025-01-31T02:52:30.686+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T02:52:30.687+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-31T02:52:30.690+0000] {logging_mixin.py:190} INFO - Files found: ['invalid_records.csv', 'valid_records.csv']
[2025-01-31T02:52:30.704+0000] {python.py:240} INFO - Done. Returned value was: validate_records_task
[2025-01-31T02:52:30.705+0000] {branch.py:38} INFO - Branch into validate_records_task
[2025-01-31T02:52:30.705+0000] {skipmixin.py:230} INFO - Following branch ('validate_records_task',)
[2025-01-31T02:52:30.719+0000] {skipmixin.py:278} INFO - Skipping tasks [('no_files_task', -1)]
[2025-01-31T02:52:30.753+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T02:52:30.754+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=validate_and_insert_csv_to_postgres_with_branch_adjusted, task_id=check_files_branch_task, run_id=manual__2025-01-31T02:52:20.519782+00:00, execution_date=20250131T025220, start_date=20250131T025230, end_date=20250131T025230
[2025-01-31T02:52:30.766+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-31T02:52:30.768+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-31T02:52:30.769+0000] {logging_mixin.py:190} INFO - Dag name:validate_and_insert_csv_to_postgres_with_branch_adjusted queued_at:2025-01-31 02:52:20.542511+00:00
[2025-01-31T02:52:30.770+0000] {logging_mixin.py:190} INFO - Task hostname:e2498a6c8083 operator:BranchPythonOperator
[2025-01-31T02:52:30.786+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T02:52:30.812+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-31T02:52:30.815+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
